### PR TITLE
Added Mystery Award example

### PR DESCRIPTION
**Pull request by aaronmatthies**
Includes example on how to use ball_holds

### CODE REVIEW

**Comment by jabdoa2** [comment]
Thanks! Looks good. Let us see if MPF is happy about the examples.

File: game_logic/ball_holds/index.rst [hunk 1]
```diff
@@ -40,7 +40,7 @@ the prefix for ball holds is ``device.ball_holds.<name>``.
 Related How To guides
 ---------------------
 
-.. todo:: :doc:`/about/help_us_to_write_it`
+:doc:`Using ball_holds for a mystery award </game_logic/mystery_award>`
```
Comment (jabdoa2) [inline game_logic/ball_holds/index.rst]: Warning, treated as error:

/home/travis/build/missionpinball/mpf-docs/game_logic/ball_holds/index.rst:43:unknown document: /game_logic/mystery_award

Comment (aaronmatthies) [reply game_logic/ball_holds/index.rst]: Sorry about the errors, it was my first time creating a page for the docs, so I'm still learning the basics. Looks like the test passed after a couple of edits.

**Comment by jabdoa2** [comment]
Awesome. Thanks for your effort!

**Pull request merged**